docs/src/pages/components/authenticator/formfields/orderformfields/form-fields.vue.mdx

```diff
@@ -0,0 +1,34 @@
+```html{3}
+<script setup>
+const formFields = {
+    signUp: {
+      email: {
+        order:1
+      },
+      family_name: {
+        order: 2
+      },
+      preferred_username: {
+        order: 4
+      },
+      birthdate: {
+        order: 3
+      },
+      password: {
+        order: 5
+      },
+      confirm_password: {
+        order: 6
+      }
+    },
+}
```

Comment:
I decided to have the custom explicitly set the order, instead of the order being whatever the order is in the `formFields` prop. This should give the customer more flexibility and allow them to update input fields in `signUp` without having to worry about affecting the order.

Reply:
👍